src/alejandra_cli/src/cli.rs

```diff
@@ -390,12 +390,16 @@ pub fn main() -> std::io::Result<()> {
 
     let formatted_paths = match matches.values_of("include") {
         Some(include) => {
-            let include = include.collect();
+            let include : Vec<_> = include.collect();
             let exclude = match matches.values_of("exclude") {
                 Some(exclude) => exclude.collect(),
                 None => vec![],
             };
 
+            if include.len() == 1 && exclude.len() == 0 && include[0] == "-" {
+                vec![crate::cli::stdin(quiet)];
```

Comment:
Something like:

```suggestion
                vec![crate::cli::stdin(quiet)]
```

or

```suggestion
                return vec![crate::cli::stdin(quiet)];
```

formatted_paths should end up being equal to a vector with the results of formatting stdin